XenAdmin/Alerts/Types/MessageAlert.cs

```diff
@@ -240,6 +240,11 @@ public override string Description
                         VMSS vmss = Helpers.XenObjectFromMessage(Message) as VMSS;
                         var policyAlertVMSS = new PolicyAlert(Message.priority, Message.name, Message.timestamp, Message.body, (vmss == null) ? "" : vmss.Name());
                         return policyAlertVMSS.Text;
+
+                    case Message.MessageType.unknown when Message.name == "GFS2_CAPACITY":

```

Comment:
I wasn't aware of the `when` keyword till now. Learn something new every day (https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/keywords/when).